def.bzl

```diff
@@ -45,6 +45,23 @@ DEFAULT_LANGUAGES = [
     Label("//language/go:go_default_library"),
 ]
 
+def _valid_env_variable_name(name):
+    """ Returns if a string is in the regex [a-zA-Z][a-zA-Z0-9_]*
+
+    Given that bazel lacks support of regex, we need to implement
+    a poor man validation
+    """
+    for character in name:
```

Comment:
Instead of looping, you can use `all(... for c in name.elems())`.

Reply:
this thing was going against buildifier linter, is the current solution good enough?